plexpy/__init__.py

```diff
@@ -280,14 +281,19 @@ def initialize_scheduler():
             seconds = 0
 
         if CONFIG.PMS_IP and CONFIG.PMS_TOKEN:
-            schedule_job(plextv.get_real_pms_url, 'Refresh Plex Server URLs', hours=12, minutes=0, seconds=0)
+            schedule_job(plextv.get_real_pms_url, 'Refresh Plex Server URLs',
+                         hours=12, minutes=0, seconds=0)
+            schedule_job(pmsconnect.get_server_friendly_name, 'Refresh Plex Server Name',
+                         hours=12, minutes=0, seconds=0)
+            schedule_job(activity_pinger.check_recently_added, 'Check for recently added items',
+                         hours=0, minutes=0, seconds=seconds)
+            schedule_job(activity_pinger.check_server_response, 'Check for server response',
```

Comment:
Thinking out loud here but why do we need to add another task to do this? We're already pinging the server to check for activity, so why not just implement it there?

For example, in activity_pinger:

```
if session_list:
    do stuff
else:
    server_connected = check_server_connectivity
    if not server_connected:
        trigger server_down
```

Those with websocket connections will fail over to the pinger, so this would work for both.